src/status_im/ui/screens/profile/styles.cljs

```diff
@@ -75,10 +82,18 @@
    :android {:font-size 12}})
 
 (defstyle profile-name-text
-  {:margin-top 8
-   :font-size  15
+  {:padding-vertical 14
+   :font-size        15
+   :text-align       :center
+   :ios              {:letter-spacing -0.2}
+   :android          {:color colors/black}})
+
+(defstyle profile-name-input-text
+  {:font-size  15
    :text-align :center
-   :ios        {:letter-spacing -0.2}
+   :ios        {:letter-spacing -0.2
```

Comment:
Use constants?

Reply:
Same thing as with colors or font sizes. I agree, but let's try to see what are the constant values and where they are used other than here.